Gemfile

```diff
@@ -3,14 +3,17 @@
 source "https://rubygems.org"
 gemspec
 
+gem "factory_bot", "~> 6.3.0"
 gem "minitest", "~> 5.11"
 gem "minitest-snapshots", "~> 1.1"
 gem "mocha", "~> 2.4"
+gem "observer"
```

Comment:
🗒️ FactoryBot 6.3 has an undeclared dependency on `observer`, which won't be part of Ruby 3.4. I had to make it an explicit dependency so that CI would pass on Ruby head.